cmd/cosign/cli/fulcio/fulcio.go

```diff
@@ -209,7 +209,7 @@ func NewClient(fulcioURL string) (api.LegacyClient, error) {
 // or a path to an identity token via the --identity-token flag
 func idToken(s string) (string, error) {
 	// If this is a valid raw token or is empty, just return it
-	if _, err := jose.ParseSigned(s); err == nil || s == "" {
+	if _, err := jwt.ParseSigned(s); err == nil || s == "" {
```

Comment:
I forgot to mention it in the PR description, but the reason I replaced `go.step.sm/crypto/jose` with `github.com/go-jose/go-jose/v3/jwt` is that the former is a wrapper around the latter (that we don't benefit from) and includes references to the `github.com/pkg/errors` package.